json/src/lib.rs

```diff
@@ -1986,6 +1986,91 @@ pub struct GetMasternodeCountResult {
     pub enabled: u32,
 }
 
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+pub struct GetMasternodeListJSON {
+    #[serde(rename = "proTxHash")]
+    pub pro_tx_hash: String,
```

Comment:
This should be decoded to a [u8;32]

Reply:
Not sure why this is needed, but the server returns a `String` data type here. As in the RPC docs:

<img width="858" alt="Screenshot 2022-07-28 at 23 33 46" src="https://user-images.githubusercontent.com/15247893/181648609-aa8ebb62-b3d1-430d-a6cc-7806e6b2fdc2.png">

<img width="812" alt="Screenshot 2022-07-28 at 23 34 01" src="https://user-images.githubusercontent.com/15247893/181648735-7bbdfdd7-ca92-4e15-b7d9-a6d6ec7c65c5.png">